src/core/reactor.ts

```diff
@@ -1734,7 +1738,7 @@ export class App extends Reactor {
                 }
     
             } else {
-                console.log(">>>>>>>>>Postponed Scheduling of timer " + timer._getFullyQualifiedName())
+                //console.log(">>>>>>>>>Postponed Scheduling of timer " + timer._getFullyQualifiedName())
```

Comment:
Maybe it's a good time to delete this logging, but I'm fine with leaving it as is.